ci/test_python_integration.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+# Support invoking test_python_singlegpu.sh outside the script directory
+cd "$(dirname "$(realpath "${BASH_SOURCE[0]}")")"/../
+
+# Common setup steps shared by Python test jobs
+source ./ci/test_python_common.sh
+
+EXITCODE=0
+trap "EXITCODE=1" ERR
+set +e
+
+rapids-logger "pytest cuml integration"
+./ci/run_cuml_integration_pytests.sh \
```

Comment:
continued naming nitpick: can we call it `run_cudf_pandas_integration_pytests.sh`? We are in the cuml repo, so "cuml integration tests" feels like a tautology, while "cudf pandas integration testing" tells me something about what we are testing. Without having to go through the layers of shell scripts to then read and understand the meaning of the option in pytest command. Three weeks ago I am not sure I'd have quickly grasped that `-p cudf.pandas` is the relevant but in the `pytest` command line and that it enables integration testing.

Reply:
That's a good point, will rename it